doc/classes/PackedByteArray.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		An array specifically designed to hold bytes. Packs data tightly, so it saves memory for large array sizes.
 		[PackedByteArray] also provides methods to encode/decode various types to/from bytes. The way values are encoded is an implementation detail and shouldn't be relied upon when interacting with external apps.
+		[b]Note:[/b] Arrays are always passed by reference. To get a copy of an array that can be modified independently of the original array, use [method duplicate]. This does [i]not[/i] apply to properties on built-in classes or return values from built-in methods, these are passed by copy, and updating the returned value will [i]not[/i] change the original value. To update a built-in property you will need to modify the returned array, and then assign it to the property again.
```

Comment:
Massive rewording suggestion, applies to every other Packed*Array.
```suggestion
		[b]Note:[/b] Packed arrays are always passed by reference. To get a copy of an array that can be modified independently of the original array, use [method duplicate]. This is [i]not[/i] the case for built-in properties and methods. The returned packed array of these is a copy, and changing it will [i]not[/i] affect the original value. To update a built-in property you need to modify the returned array, and then assign it to the property again.
```
I get you tried to be as explicit as possible before, but the proper note in the class reference does a much better job at this, now.

Reply:
Thank you will take a look 🙂

Reply:
Not sure if this is desired or not but I'm going to comment on it as a reminder

Reply:
Will see about adding this when I get back to looking at the specifics, just resolved some conflicts yesterday